docs/pages/use_cases/COMPSs.md

```diff
@@ -27,7 +27,7 @@ roles: [information_architect, researcher, software_developer] # should match it
 
 # COMPSs
 
-[![COMPSs logo](../../assets/img/COMPSs.png)](https://compss.bsc.es/)
+[![COMPSs logo](assets/img/COMPSs.png)](https://compss.bsc.es/)
```

Comment:
Check which style as #322  also fixed this but to `../../` variant

Reply:
Both seem to work locally, but `assets/img/COMPSs.png` definitely works in production and it's what we've used elsewhere